udata/forms/fields.py

```diff
@@ -200,10 +198,29 @@ def __getattr__(self, name):
 
 class FormField(FieldHelper, fields.FormField):
     def __init__(self, form_class, *args, **kwargs):
+        self.has_data = False
```

Comment:
I'm not familiar enough with WTForm but that looks weird that you have to deal with that manually :/